Jenkinsfile

```diff
@@ -8,11 +8,31 @@ node('us-east-1 && ubuntu && docker && !gpu') {
         checkout scm
         def delim = '"'
         hash = sh(returnStdout: true, script: """curl https://api.github.com/repos/cliqz-oss/cliqz-android/branches/upstream-releases | grep -m 1 sha | cut -d '${delim}' -f 4""").trim()
+        withCredentials([file(credentialsId: 'ceb2d5e9-fc88-418f-aa65-ce0e0d2a7ea1', variable: 'CLIQZ_CI_SSH_KEY')]) {
+            sh '''#!/bin/bash -l
+                set -x
+                set -e
+                mkdir -p ~/.ssh
+                cp $CLIQZ_CI_SSH_KEY ~/.ssh/id_rsa
+                chmod 600 ~/.ssh/id_rsa
+                echo $CLIQZ_CI_SSH_KEY
+                ssh-keyscan -t rsa github.com >> ~/.ssh/known_hosts
+                git clone -b version2.0 --single-branch --depth=1 git@github.com:cliqz/autobots.git
+            '''
+        }
     }
     def dockerTag = "${hash}"
     def baseImageName = "browser-f/android:${dockerTag}"
-    docker.image("141047255820.dkr.ecr.us-east-1.amazonaws.com/${baseImageName}").inside {
-        try {
+    def instance_id = sh(returnStdout: true, script: '''
+            aws ec2 run-instances --image-id ami-13050368 --count 1 --instance-type t2.medium --key-name android_ci_genymotion --security-group-ids sg-5bbf173f --subnet-id subnet-341ff61f  --region=us-east-1 --query "Instances[].InstanceId" --output text
```

Comment:
how about splitting this long string into multiple lines?

Reply:
Done.